### PR TITLE
Update NodeJS to version 20

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: ops/roles/infra/node/tasks/main.yml [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-- name: Update Npm to latest version
-  command: npm install -g npm
+- name: Install NodeJS and NPM
+  ansible.builtin.apt:
+    name: nodejs
+    update_cache: true
+    state: latest # The major version of NodeJS is provided by the NodeSource repository defined in the task above
 
-- name: Install pm2
-  command: npm install -g pm2 --production=true
```
Comment (MattiSG) [inline ops/roles/infra/node/tasks/main.yml:21]: Why was this removed? Is this now working by coincidence because `pm2` is already installed as a leftover?

Comment (Ndpnt) [reply ops/roles/infra/node/tasks/main.yml:21]: That's a good point! Here is the fix: https://github.com/OpenTermsArchive/contribution-tool/pull/157
